{{cookiecutter.repo_name}}/pyproject.toml

```
@@ -0,0 +1,45 @@
+[build-system]
+requires = [
+  "setuptools >= 40.9.0",
+  "versioneer-518",
```

```suggestion
  "versioneer",
```
? maybe I'm missing something about the versioneer-518 package, I thought this was the right pypi location? https://pypi.org/project/versioneer/

---

I think versioneer currently may not play well with pyproject.toml, so -518 (for PEP518) there's a special version https://github.com/python-versioneer/python-versioneer/issues/193#issue-408237852 . We could also re-think using versioneer, although I've not personally had problems with it

---

So I'm not sure I fully understand the way that package is intended to work here. As far as I could understand from that thread, that's specifically for cases where you want to not vendor versioneer? I assumed (maybe incorrectly?) we would have vendored here, are we not?

---

Yup, i think so. i removed `versioneer` from pyproject.toml since I think that still pulls in the wrong one